libraries/AP_HAL_ChibiOS/hwdef/BROTHERHOBBYH743/hwdef.dat

```diff
@@ -0,0 +1,196 @@
+# USB
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+PE2 VBUS INPUT OPENDRAIN
+
+PA13 JTMS-SWDIO SWD
+PA14 JTCK-SWCLK SWD
+
+# SPI1 for IMU1 (ICM42688)
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PD7 SPI1_MOSI SPI1
+PC15 IMU1_CS CS
+
+# SPI2 for MAX7456 OSD
+PB12 MAX7456_CS CS
+PB13 SPI2_SCK SPI2
+PB14 SPI2_MISO SPI2
+PB15 SPI2_MOSI SPI2
+
+# SPI3 - external
+PB3 SPI3_SCK SPI3
+PB4 SPI3_MISO SPI3
+PB5 SPI3_MOSI SPI3
+PD4 EXT_CS1 CS
+
+# SPI4 for IMU2 (ICM42688)
+PE11 IMU2_CS CS
+PE12 SPI4_SCK SPI4
+PE13 SPI4_MISO SPI4
+PE14 SPI4_MOSI SPI4
+
+# two I2C bus
+I2C_ORDER I2C2 I2C1
+
+# I2C1
+PB6 I2C1_SCL I2C1
+PB7 I2C1_SDA I2C1
+
+# I2C2
+PB10 I2C2_SCL I2C2
+PB11 I2C2_SDA I2C2
+
+# ADC
+PC0 BATT_VOLTAGE_SENS ADC1 SCALE(1)
+PC1 BATT_CURRENT_SENS ADC1 SCALE(1)
+
+define HAL_BATT_MONITOR_DEFAULT 4
+define HAL_BATT_VOLT_PIN 10
+define HAL_BATT_CURR_PIN 11
+define HAL_BATT_VOLT_SCALE 11.0
+define HAL_BATT_CURR_SCALE 40.0
+
+PC5 RSSI_ADC ADC1
+define BOARD_RSSI_ANA_PIN 8
+
+# LED
+# green LED1 marked as B/E
+# blue LED0 marked as ACT
+define AP_NOTIFY_GPIO_LED_2_ENABLED 1
+PE9 LED0 OUTPUT LOW GPIO(90) # blue
+PA7 LED1 OUTPUT LOW GPIO(91) # green
+define HAL_GPIO_A_LED_PIN 91
+define HAL_GPIO_B_LED_PIN 90
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 EMPTY USART6 UART7 UART8 OTG2
+
+# USART1 (ESC Telemetry)
+PA10 USART1_RX USART1 NODMA
+PA9  USART1_TX USART1 NODMA
+define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_ESCTelemetry
+
+# USART2 (RX/SBUS)
+PD5 USART2_TX USART2
+PD6 USART2_RX USART2
+define DEFAULT_SERIAL2_PROTOCOL SerialProtocol_RCIN
+
+# USART3 (spare)
+PD9 USART3_RX USART3
+PD8 USART3_TX USART3
+
+# UART4
+PB9 UART4_TX UART4 NODMA
+PB8 UART4_RX UART4 NODMA
+
+# USART6 (DJI)
+PC7 USART6_RX USART6
+PC6 USART6_TX USART6
+define DEFAULT_SERIAL6_PROTOCOL SerialProtocol_MSP_DisplayPort
+
+# UART7 (GPS)
+PE7 UART7_RX UART7
+PE8 UART7_TX UART7
+define DEFAULT_SERIAL7_PROTOCOL SerialProtocol_GPS
+
+# UART8 (spare)
+PE0 UART8_RX UART8 NODMA
+PE1 UART8_TX UART8 NODMA
+
+# CAN bus
+PD0 CAN1_RX CAN1
+PD1 CAN1_TX CAN1
+PD3 GPIO_CAN1_SILENT OUTPUT PUSHPULL SPEED_LOW LOW GPIO(70)
+
+# Motors
+PB0  TIM3_CH3 TIM3  PWM(1)  GPIO(50) BIDIR
+PB1  TIM3_CH4 TIM3  PWM(2)  GPIO(51)
+PA0  TIM2_CH1 TIM2  PWM(3)  GPIO(52) BIDIR
+PA1  TIM2_CH2 TIM2  PWM(4)  GPIO(53)
+PA2  TIM5_CH3 TIM5  PWM(5)  GPIO(54) BIDIR
+PA3  TIM5_CH4 TIM5  PWM(6)  GPIO(55)
+PD12 TIM4_CH1 TIM4  PWM(7)  GPIO(56) BIDIR
+PD13 TIM4_CH2 TIM4  PWM(8)  GPIO(57)
+PD14 TIM4_CH3 TIM4  PWM(9)  GPIO(58)
+PD15 TIM4_CH4 TIM4  PWM(10) GPIO(59)
+
+# Disable DMA on PWM11-12 so that the LEDs get a channel
+PE5  TIM15_CH1 TIM15 PWM(11) GPIO(60) NODMA # SERVO 1
+PE6  TIM15_CH2 TIM15 PWM(12) GPIO(61) NODMA # SERVO 2
+
+PA8 TIM1_CH1 TIM1 PWM(13) GPIO(62) # for WS2812 LED
+
+# Beeper
+PA15 BUZZER OUTPUT GPIO(32) LOW
+define HAL_BUZZER_PIN 32
+
+# microSD support
+PC8 SDMMC1_D0 SDMMC1
+PC9 SDMMC1_D1 SDMMC1
+PC10 SDMMC1_D2 SDMMC1
+PC11 SDMMC1_D3 SDMMC1
+PC12 SDMMC1_CK SDMMC1
+PD2 SDMMC1_CMD SDMMC1
+
+# GPIOs
```

Comment:
are these camera control and vtx power?dont see them in pinout,...if so need hwdef entries and section in readme